CHANGELOG.md

```diff
@@ -9,13 +9,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
-* Prepared statements implemented along with a string cache for the queries. This will speed up many repeated database calls.
+* Prepared statements implemented along with a string cache for the queries.
```

Comment:
These are unreleased changes being prepared for release, so yes they are liable to change. In this case reducing detail, since it does not give the reader anything useful. The CHANGELOG is supposed to be brief and human readable.